internal/filter/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: [open-telemetry/collector-approvers]
```

Comment:
And this?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/.github/CODEOWNERS#L113
```
internal/filter/                                         @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
```